.editorconfig

```diff
@@ -0,0 +1,20 @@
+# Shopware 5 editor configuration normalization
```

Comment:
Shopware5?

Reply:
c&p from other frosh plugins, because a some of files had no new line at the end and some tabs mixed with sapces.